04/app.js

```diff
@@ -0,0 +1,30 @@
+
+const divsList = document.querySelectorAll('div');
+const bodyEl = document.querySelector('body');
+
+
+const addClassToElement = function (e) {
+
+    if (e.toElement.className === 'child') {
```

Comment:
Rozumiem, że to było zabezpieczenie przed tym, aby kliknięcie w `body` nie powodowało kolejnego uruchomienia.